http/engine.go

```diff
@@ -330,7 +330,7 @@ func (h Engine) applyBindMiddleware(echoServer EchoServer, path string, excludeP
 		log.Logger().Infof("Enabling token authentication for HTTP interface: %s%s", address, path)
 		echoServer.Use(middleware.JWTWithConfig(middleware.JWTConfig{
 			KeyFunc: func(_ *jwt.Token) (interface{}, error) {
```

Comment:
Should be changed for `ParseTokenFunc` which is passed a ctx. Link an issue if not for this PR.

Reply:
Not really possible since ParseTokenFunc can't return the signing key and is not supposed to validate it. "Fixed" it by moving the signing key fetching to node initialization phase. Then it also happens only once, instead of every time a token is validated.